modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: CONTEXT
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Using `oc adm upgrade status` to retrieve information about a cluster update (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
```

Comment:
One thing we probably need to stress out is that because this command is a Technology Preview, it has to be explicitly enabled by setting the `OC_ENABLE_CMD_UPGRADE_STATUS` environmental variable to `true`, otherwise it will not work and oc will behave as if no such command exists.